### PR TITLE
Add storage opts

### PR DESCRIPTION
**Pull request by bennetty**
add storage opts flag

### CODE REVIEW

**Comment by nithyaasworld** [comment]
Thanks @bennetty for the PR! Sorry that it took us a long time to get to this. We verified it and we would be merging this soon. 

Could you also please update the Readme file with an example for this? Thanks again!

**Comment by nithyaasworld** [comment]
While the plugin works, the [storage-opts flag](https://docs.docker.com/engine/reference/commandline/run/#set-storage-driver-options-per-container) is added as an array though it supposed to have only one value. Closing this PR. @bennetty  If you would like, please feel free to re-open this PR with updated changes that would accept storage-opts flag as string rather than an array.

File: plugin.yml [hunk 1]
```diff
@@ -65,6 +65,8 @@ configuration:
       type: boolean
     sysctls:
       type: array
+    storage-opt:
+      type: string
```
Comment (pzeballos) [inline plugin.yml:69]: Makes sense! 👌
We probably should order attributes in alphabetical order ... but that's outside this PR

File: README.md [hunk 2]
```diff
@@ -231,6 +231,12 @@ Set the size of the `/dev/shm` shared memory filesystem mount inside the docker
 
 Example: `2gb`
 
+### `storage-opt` (optional, string)
+
+This allows to set the container rootfs size at the creation time. This is only available for the devicemapper, btrfs, overlay2, windowsfilter and zfs graph drivers. See [docker documentation](https://docs.docker.com/engine/reference/commandline/run/#set-storage-driver-options-per-container) for more details.
```
Comment (pzeballos) [inline README.md:236]: I would add a backtick on each driver name:

```suggestion
This allows setting the container rootfs size at the creation time. This is only available for the `devicemapper`, `btrfs`, `overlay2`, `windowsfilter` and `zfs` graph drivers. See [docker documentation](https://docs.docker.com/engine/reference/commandline/run/#set-storage-driver-options-per-container) for more details.
```

**Review by pzeballos — CHANGES_REQUESTED** [review]
Only a small change on the README, but the rest LGTM

**Review by pzeballos — APPROVED** [review]

**Comment by pzeballos** [comment]
Thanks @bennetty for your contributions and sorry for the delay!!

**Pull request merged**